pkg/connectors/datacatalog/clients/datacatalog_grpc.go

```diff
@@ -13,6 +13,7 @@ import (
 	"emperror.dev/errors"
```

Comment:
Not related to this PR. However, when will grpc support be removed?

Reply:
First we need to list where all we have used grpc. Probably we can raise a git issue first enlisting the various parts of the source which needs to be removed. Then remove it in a PR. Does this sound ok? @shlomitk1